benchmark/parse.php

```diff
@@ -1,7 +1,12 @@
 <?php
 
-require __DIR__ . '/../vendor/autoload.php';
+require __DIR__.'/../vendor/autoload.php';
 
+$components = [
+    'scheme' => 'https',
+    'host' => 'uri.thephpleague.com',
+    'path' => '/5.0',
+];
 for ($i = 0; $i < 100000; $i++) {
-    League\Uri\Http::createFromString('http://amphp.org/amp');
+    League\Uri\Http::createFromComponents($components);
```

Comment:
It no longer benchmarks the parsing now, so the file should be renamed.

Reply:
how would you call the file  ? simply `benchmark.php` ?

Reply:
Maybe simply `http-create-from-components.php`?